docs/source/tutorials/running_a_simulation/cli.rst

```diff
@@ -0,0 +1,120 @@
+.. _cli_tutorial:
+
+=============================
+Running from the Command Line
+=============================
+
+To run from the command line, we'll use the
+:func:`vivarium.interface.cli.simulate` command. This command is actually a
+group containing three sub-commands: ``run``, ``test``, and ``profile``. We will
+focus on ``run`` here.
+
+The basic use of ``simulate run`` requires no more than a
+:term:`model specification <Model Specification>` yaml file. With this, you can
+do the following to run the model defined by that specification:
+
+.. code-block:: console
+
+    simulate run /path/to/your/model/specification.yaml
+
+By default, ``simulate run`` will output whatever results your model produces
+to the ``~/vivarium_results`` directory.
+
+.. note::
+
+   ``~`` is used as a shortcut to represent the user's home directory on
+   a file system. If you're on Windows, this probably looks something like
+   ``C:\Users\YOUR_NAME`` while on linux it would be ``/home/YOUR_NAME``.
+
+If you navigate to that directory, you should see a subdirectory with the name
+of your model specification.  Inside the model specification directory, there
+will be another subdirectory named for the start time of the run. In here, you
+should see two hdf files: ``final_state.hdf``, which is the population
+:term:`state table <State Table>` at the end of the simulation, and
+``output.hdf``, which is the results of the :term:`metrics <Metrics>` generated
+by the simulation.
+
+For example, say we've run a simulation for a model specification called
+``potatoes.yaml`` (maybe we're really into gardening).  Our directory tree
```

Comment:
lol love the potatoes example

Reply:
Have to keep @alec-deason 's legacy alive.